modules/security-context-constraints-psa-synchronization.adoc

```diff
@@ -18,18 +18,14 @@ Applying pods directly might use the SCC privileges of the user who runs the pod
 
 [IMPORTANT]
 ====
-Namespaces that have an `openshift-` name prefix and were not created by the system during the installation are not synchronized by default.
-
-Namespaces that have the `openshift-` prefix are typically system namespaces; by convention, a controller should exist to manage them.
-
-You can enable SCC synchronization in namespaces that have the `openshift-` prefix by setting the value of the `security.openshift.io/scc.podSecurityLabelSync` label to `true`.
-
 Namespaces that are defined as part of the cluster payload have pod security admission synchronization disabled permanently. These namespaces include:
 
-* All namespaces that are prefixed with `openshift-`, except for `openshift-operators`
 * `default`
 * `kube-node-lease`
 * `kube-system`
 * `kube-public`
 * `openshift`
+* All system-created namespaces that are prefixed with `openshift-`, except for `openshift-operators`
+
+By default, all namespaces that have an `openshift-` prefix are not synchronized. You can enable synchronization for any user-created [x-]`openshift-*` namespaces by setting the value of the `security.openshift.io/scc.podSecurityLabelSync` label to `true`. You cannot enable synchronization for any system-created [x-]`openshift-*` namespaces, except for `openshift-operators`.
```

Comment:
```suggestion
By default, {product-title} will not synchronize namespaces that have an `openshift-` prefix. 
```
Does it make sense to reword this to avoid passive voice?

Reply:
The original passive voice still sounds the best to me here